extensions/molecule/analysis/converge.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: Test pre-conversion analysis
+  ansible.builtin.import_playbook: infra.convert2rhel.analysis
```

Comment:
locally it's failing for me because it can't import it from this path, am I expected to have it there?

Reply:
Heh, this is what we were talking about yesterday... You have to place the project in a very specific project structure to make it work 😓 

First off, the base folder should be `ansible_collections`, and then the `infra.convert2rhel` folder needs to be split in the format `infra/convert2rhel`. 

After that, you have to change the variable ANSIBLE_COLLECTIONS_PATH in the Makefile to point to the path before your `ansible_collections` folder, for example: 

```bash
ANSIBLE_COLLECTIONS_PATH=/home/awaltlov/documents/work_repos make analysis
```

Assuming, of course, that you will have this setup: `/home/awaltlov/documents/work_repos/ansible_collections/infra/convert2rhel`

Reply:
I will update the docs later.

Reply:
huh I still don't get it .. I did `make`

```
create-folder-for-test:
    mkdir -p ansible_collections/infra/convert2rhel/
    cp playbooks/* ansible_collections/infra/convert2rhel/
```

and then tried to run `make analysis` with pwd (`ANSIBLE_COLLECTIONS_PATH:=$(shell pwd)`) as parent of the newly created `ansible_collections` folder, but still same error

I will approve it, probably the problem is me :sweat_smile:

Reply:
Alright works with

```
ANSIBLE_COLLECTIONS_PATH:=$(shell pwd)
...

create-folder-for-test:
	mkdir -p ansible_collections/infra/convert2rhel/
	cp -r playbooks plugins roles meta galaxy.yml ansible_collections/infra/convert2rhel/

....

analysis: install-deps create-folder-for-test
	. $(PYTHON_VENV)/bin/activate; cd extensions && molecule converge --scenario-name analysis
	rm -rf ansible_collections/
```